foreman-proxy/foreman-proxy.service

```diff
@@ -8,6 +8,7 @@ Type=forking
 User=foreman-proxy
 ExecStart=/usr/share/foreman-proxy/bin/smart-proxy
 PIDFile=/run/foreman-proxy/foreman-proxy.pid
+TimeoutSec=infinity
```

Comment:
This should probably be `TimeoutStartSec` only, since the bug probably doesn't affect shutdown - disabling the timeout feature on shutdown will prevent systemd killing the service more abruptly when it fails.